components/tikv_util/src/future.rs

```diff
@@ -72,25 +74,152 @@ where
 /// thread calling `wake()`.
```

Comment:
```suggestion
/// thread calling `poll_future_notify()`.
```

Reply:
`wake` is correct. It's polled in `wake`.

Reply:
I understand the comment now.